node/src/main/java/io/airlift/node/NodeConfig.java

```diff
@@ -169,4 +164,21 @@ public NodeConfig setConfigSpec(String configSpec)
         this.configSpec = configSpec;
         return this;
     }
+
+    public AddressType getAddressType()
+    {
+        return addressType;
+    }
+
+    @Config("node.internal-address-type")
```

Comment:
Maybe "internal-address-lookup-method"? Also, I can understand "find ip from interfaces" vs "use hostname", but why do we need hostname vs fqdn?